nordicNRPlus/thingExists.ts

```diff
@@ -0,0 +1,17 @@
+import { type IoTClient, DescribeThingCommand } from '@aws-sdk/client-iot'
+
+export const thingExists = async (
+	iotClient: IoTClient,
+	thingName: string,
+): Promise<boolean> => {
+	try {
+		await iotClient.send(new DescribeThingCommand({ thingName }))
+		return true
+	} catch (err: any) {
+		if (err.name === 'ResourceNotFoundException') {
+			return false
+		} else {
+			return false
+		}
```

Comment:
```suggestion
		return false
```

Both paths return the same.